browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -18,7 +18,6 @@
 #include "brave/components/constants/brave_switches.h"
 #include "build/build_config.h"
 #include "chrome/browser/browser_process.h"
-#include "chrome/browser/net/system_network_context_manager.h"
```

Comment:
This was needed for `SystemNetworkContextManager` due to `chrome/browser/browser_process.h` forward declaring `SystemNetworkContextManager`

Reply:
`SystemNetworkContextManager` is no longer used in this casee

Reply:
Ah, strange as I was looking at `g_browser_process->system_network_context_manager()
                ->GetSharedURLLoaderFactory()` which has clearly being removed :-)